src/iuse_actor.cpp

```diff
@@ -1093,8 +1093,8 @@ int salvage_actor::cut_up(player *p, item *it, item *cut) const
         int amount = salvaged.second;
         item result( mat_name, int(calendar::turn) );
         if (amount > 0) {
-            add_msg( m_good, ngettext("Salvaged %1$i %2$s.", "Salvaged %1$i %2$ss.", amount),
-                     amount, result.display_name().c_str() );
+            add_msg( m_good, ngettext("Salvaged %1$i %2$s.", "Salvaged %1$i %2$s.", amount),
```

Comment:
The verb could be different in some language based on whether it refers to one or several things. No idea whether it's actually used in any language.